script/Ciel/Day 7/07_01_CIEL07_1_1.txt

```diff
@@ -27,11 +27,11 @@ Of course, just ground meat alone would leave my diet a bit unbalanced,
 [sha:c005a5403bf12bac63e89d419495cd1d9731a184]{
 -- Page 86, Offset 15008.
 -- ん？　と引っかかるものがあったが食欲には勝てない。
-Something tugs at the back of my mind at that statement, but it doesn't overpower my appetite.
+Something in what he just said tugs at the back of my mind, but it isn't enough to overpower my appetite.
 }
 [sha:b25e31cc4a47415f834ffa896954b2928916ae1a]{
 -- Page 86, Offset 15009.
 -- 俺は絶好のチャンスを棒に振って、男友達と仲良くランチに行くのだった。
-// 男友達 could maybe be emphasized (as in, he's going eating with a dude when he could be eating with Ciel)
-And so, ruining a perfect chance, I instead go eating lunch together with my pal.
+// 男 split off of 友達 to emp lack of Ciel without having Shiki say 'broski' or something
+And so, letting a perfect opportunity go to waste, I go for a lads lunch with my good friend.
```

Comment:
"lads lunch" is not very american but I'll let it slide this once because it is pretty fire